features/generate.feature

```diff
@@ -27,6 +27,17 @@ Feature: Generate a new project using Tuist
     Then I should be able to build the scheme Framework1
     Then the product 'Framework1.framework' with destination 'Debug-iphoneos' contains the Info.plist key 'Test'
 
+  Scenario: The project is an iOS application with tests (app_with_framework_and_tests_with_custom_filenames)
```

Comment:
Since acceptance tests are slow, I'd use this feature in one of the existing project and reuse one of the existing tests.